yaml/places.yaml

```diff
@@ -1,4 +1,7 @@
 global:
+  - &ne_country_capital [Admin-0 capital, Admin-0 capital alt, Admin-0 region capital]
```

Comment:
Please use this for setting `ne_country_capital`, too :)

Great implementation!

Reply:
Good idea. Added in c87a6b0d.